k8s/openebs-operator.yaml

```diff
@@ -36,7 +36,10 @@ rules:
   resources: ["customresourcedefinitions"]
   verbs: [ "get", "list", "create" ]
 - apiGroups: ["*"]
-  resources: ["storagepools"]
+  resources: [ "cstorpools","cstorvolumereplicas","cstorvolumes" ]
+  verbs: ["*" ]
+- apiGroups: ["*"]
+  resources: ["castemplates"]
```

Comment:
As kiran said, all the CR's we can defined in same place unless they have different verbs/actions ? for example : 
```
- apiGroups: ["*"]
  resources: ["storagepoolclaims","disks","storagepools", "cstorpools","cstorvolumereplicas","cstorvolumes" ]
  verbs: ["get", "list",etc-----,]
```